src/Plugin.php

```diff
@@ -85,7 +86,8 @@ public function __invoke(RegistrationInterface $api, SimpleXMLElement $config =
         $api->registerHooksFromClass(ContainerHandler::class);
 
         $stubs = array_merge(
-            $this->getCommonStubs(), $this->getStubsForMajorVersion(Kernel::MAJOR_VERSION)
+            $this->getCommonStubs(),
+            $this->getStubsForMajorVersion(Kernel::MAJOR_VERSION),
```

Comment:
```suggestion
            $this->getStubsForMajorVersion(Kernel::MAJOR_VERSION)
```
tests for lower versions are failing because of this

Reply:
**Thank you**, I honestly would not notice that. Fixed and commited.